features/src/icc/install.sh

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+set -ex
+
+ICC_VERSION="${VERSION:-$DEFAULT}";
```

Comment:
what is `$DEFAULT` here?

Reply:
So this was my guess:

It's the default listed in the feature json. I thought it would get defined in the env for the script.